.stylelintrc

```diff
@@ -4,7 +4,7 @@
   ],
   "rules": {
     "at-rule-empty-line-before": ["always", {
-      "except": ["blockless-group", "first-nested"],
```

Comment:
You're missing the `blockless-after-same-name-blockless` for the sequencial `@import` expressions.